src/components/shop/shop-list-screen.js

```diff
@@ -11,18 +11,15 @@ import { onScaleChange } from "../../core/scaler.js";
 import { createTitles } from "../../core/titles.js";
 import { createBalance } from "./balance-ui.js";
 import { gmi } from "../../core/gmi/gmi.js";
+import { initResizers } from "./backgrounds.js";
 
 export class ShopList extends Screen {
     preload() {
         this.plugins.installScenePlugin("rexUI", RexUIPlugin, "rexUI", this, true);
+        initResizers();
     }
 
     create() {
-        if (this.transientData.shop.mode === "shop") {
-            gmi.sendStatsEvent("shopbuy", "click", {});
-        } else {
-            gmi.sendStatsEvent("shopmanage", "click", {});
-        }
```

Comment:
we need this stat right?

Reply:
hm did this not fail a unit test either

Reply:
That's kind of strange. I never touched that. Will revert and check.

Reply:
Added back in but made it a one-liner.